docs/team/member-guide.md

```diff
@@ -68,6 +68,9 @@ a few things that you should do as a new team member:
    membership requires commitment from both sides. Beyond 6 months, if active
    team membership is no longer possible for you, you can always
    join the green team!
+5. **Let us know if you'll be unavailable** or out of town for an extended period
```

Comment:
What's the process for letting us know? Email, issue, gitter, announcement at the team meeting?

Reply:
Good question - what about something like "Open an issue in the team-compass if you'll be away, and if it's something you'd rather not mention to the public then send an email to one of the team members and they can communicate it to the others"

?

Reply:
Sounds good to me :)

Reply:
done!

Reply:
What counts as an extended period? A one or two week off-grid holiday, or more like a month?